compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/interop/JMethodResolver.java

```diff
@@ -897,6 +898,11 @@ private Executable resolveMethod(Class<?> clazz, String name, Class<?>... paramT
     private List<Executable> getExecutables(Class<?> clazz, String methodName, JMethodKind kind) {
 
         if (kind == JMethodKind.CONSTRUCTOR) {
+            if (Modifier.isAbstract(clazz.getModifiers())) {
+                String classType = clazz.isInterface() ? "interface" : "abstract class";
+                throw new JInteropException(DiagnosticErrorCode.INSTANTIATION_ERROR,
+                        "Cannot instantiate " + classType + " '" + clazz.getName() + "'");
```

Comment:
```suggestion
                        "'" + clazz.getName() + "' is abstract; cannot be instantiated");
```
We can say like this for both the interfaces and abstract classes.

Reply:
Then we can remove the line 902 also.

Reply:
Is the usual format we use in ballerina compilation errors?

Reply:
Adapting this suggestion since Java also provides the same compilation errors.